CacheWarmer/AspectCacheWarmer.php

```diff
@@ -94,7 +93,7 @@ public function warmUp($cacheDir)
                     "/resource=" . $realPath
                 );
             } catch (\Exception $e) {
-                $errors[$realPath] = $e;
+                /* noop */
```

Comment:
Why do you decided to silence all exceptions? Shouldn't we report about errors in verbose or debug mode?